### PR TITLE
task-Dt4HutPs 

### PR DESCRIPTION
**Pull request by AntonShalenui**
https://trello.com/c/Dt4HutPs/27-connect-maincss

### CODE REVIEW

File: src/client/components/Main.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default class Main extends Component {
     render() {
         const { username } = this.state;
         return (
-          <main className = "main-text">
+          <main className ="main-text">
```
Comment (vkryviak-ua) [inline src/client/components/Main.js]: Why did you remove whitespace? Please remove both of them or noone (className="main-text" or className = "main-text")

**Comment by vkryviak-ua** [comment]
And please do not forget add description like a `Added main.css to Main.js`

File: src/client/components/Main.js (hunk 1 above)
Comment (AntonShalenui) [reply src/client/components/Main.js]: because I don't know how to do this correct
